# Module 67.1 Study - Mortgage-Backed Security (MBS) Instrument and Market Features

<hr>


### LOS 67.a: Define prepayment risk and describe time tranching structures in securitizations and their purpose.

An important characteristic of investments in mortgage-backed securities (MBSs) is their **prepayment risk**. Prepayments are principal repayments by mortgage borrowers in excess of the scheduled principal repayments for amortizing loans. In a pool of mortgages, some of the borrowers are likely to prepay, typically because they sell their homes or refinance their mortgages.

MBS valuation is based on an assumed prepayment rate for the underlying mortgages. Prepayment risk is the risk that prepayment speeds turn out to be *different to the expectations* of MBS investors when they purchased the MBS.

The risk that prepayments will be slower than expected, leading to MBS investors waiting longer than originally anticipated for their cash flows, is called **extension risk**. The risk that prepayments will be faster than expected, leading to cash flows arriving sooner than expected, is called **contraction risk**.

A key driver of prepayment speeds is the prevailing level of interest rates. When interest rates decrease, borrowers often refinance their mortgages at lower rates. This will cause the original higher-rate mortgages to be repaid early, increasing prepayment speeds and leading to contraction in the average life of an MBS. This is bad for MBS investors for two reasons:

* They receive cash flows sooner than expected in a low-rate environment and face lower reinvestment returns.
* Because the prices of MBS reflect expectations for prepayments in low-rate environments, they will not rise as much in response to decreasing interest rates as other fixed-income instruments that do not have an embedded prepayment option.


>**Professor's Note**
\
MBSs are like callable bonds, where the borrower has the right to repay the loan early, which leads to negative convexity for prices at low yields (i.e., prices rise at a slower rate as yields fall).

When interest rates increase, borrowers' prepayment speeds will be slower than expected because refinancing activity will slow, leading to extension in the average life of an MBS. This is also bad for MBS investors because cash flows will be discounted by more periods at a higher discount rate.

One way to reapportion the prepayment risk inherent in the underlying mortgage pool is to use **time tranching** in the MBS structure. An MBS with time tranching will have different bond classes with different maturities. Contraction and extension risk still exist with this structure, but the risks are redistributed to some extent among the tranches. The tranches that mature first offer investors protection against extension risk. Tranches with longer maturities provide relatively more protection against contraction risk. We will describe one method of time tranching, called sequential pay tranching, later in this reading.

### LOS 67.b: Describe fundamental features of residential mortgage loans that are securitized.

A **residential mortgage loan** is a loan for which the collateral that underlies the loan is residential real estate. If the borrower defaults on the loan, the lender has a legal claim to the collateral property (referred to as a first lien), whereby they can take possession of the property and sell it to recover losses (a process called foreclosure).

>**Professor's Note**
\
MBSs backed by residential mortgages are called residential mortgage-backed securities (RMBSs). These residential mortgages are the focus of this LOS. Commercial mortgages, which relate to investments in properties to generate income rather than to provide a home for the borrower, as well as their related commercial mortgage-backed securities, are discussed later in the reading.

Common features of residential mortgage loans include the following:

* *Prepayment penalties*. A prepayment penalty is an additional payment that must be made by borrowers to compensate lenders if principal is prepaid when interest rates decline. Common in Europe, but rare in the United States, these penalties are designed to reduce prepayment risk for lenders.

* *Recourse/nonrecourse loans*. Nonrecourse loans only have the specified property as collateral. With recourse loans, the lender also has a claim against other assets of the borrower for any amount by which the foreclosure falls short of the payments outstanding on a defaulting loan. When the property value falls below the outstanding mortgage balance (a situation known as underwater mortgages or negative equity), borrowers with nonrecourse loans are more likely to strategically default, preferring to face foreclosure and a reduction in their credit score rather than continue to make payments on a loan exceeding the value of the property. In Europe, most residential mortgages are recourse loans. In the United States, the nature of the loans varies by state, but most loans are nonrecourse.

#### Loan-to-Value and Debt-to-Income Ratios

A key measure of default risk for a mortgage loan is its **loan-to-value ratio (LTV)**, the percentage of the value of the collateral real estate that is loaned to the borrower. The lower the LTV, the higher the borrower's equity in the property. For a lender, loans with lower LTVs are less risky because the borrower has more to lose in the event of default (so is less likely to default). Also, if the property value is high compared to the loan amount, the lender is more likely to recover the amount loaned if the borrower defaults and the lender repossesses and sells the property.

Another key measure is the borrower's **debt-to-income ratio (DTI)**, which measures the monthly debt payments of the individual as a percentage of their monthly pretax gross income. A borrower with a lower DTI is deemed of lower risk to default.

For example, a borrower wishing to take out a £300,000 mortgage on a £400,000 property will have an `LTV` of `£300,000 / £400,000 = 75%`.

If the mortgage has an annual interest rate of 6%, is to be repaid monthly over 25 years, and the borrower's annual pretax gross income is £80,000, then we can calculate the DTI as follows.

First, calculate the constant monthly payment required to service the mortgage:

    N = 25 × 12 = 300; I/Y = 6 / 12 = 0.5; PV = 300,000; FV = 0; CPT → PMT = £1,932.90

Second, calculate the monthly pretax gross income of the borrower as 
\
`£80,000 / 12 = £6,667.`
\
Then, calculate `DTI` as `£1,932.90 / £6,667 = 29%`.

In the United States, mortgages made to borrowers with good credit, low LTV, and low DTI are termed prime loans. Mortgages to borrowers of lower credit quality, with higher DTI, higher LTV, or that have a lower-priority claim to the collateral in event of default, are termed **subprime loans**.

#### Agency and Non-Agency RMBSs

In the United States, a distinction is made between agency and non-agency RMBSs.

**Agency RMBSs** are either guaranteed by the government or guaranteed by a government-sponsored enterprise (GSE), which are companies created by the government. Because GSEs are technically distinct from the government, the securities guaranteed by GSEs do not carry the full faith and credit of the government—only that of the GSE that provides the guarantee. Mortgages need to meet minimum underwriting standards to qualify as collateral for an agency RMBS.

**Non-agency RMBSs** are issued by private entities such as banks and have no government or GSE guarantee. Non-agency RMBSs typically include credit enhancements through external insurance, letters of credit, tranching, and private guarantees. Non-agency RMBSs are often cited as a catalyst for the global financial crisis of 2007–2009, when investment-grade senior tranches backed by subprime mortgages suffered large losses due to significant defaults in the underlying collateral. Issuance of non-agency RMBSs effectively ceased after the credit crisis due to changes in regulation.

### LOS 67.c: Describe types and characteristics of residential mortgage-backed securities, including mortgage pass-through securities and collateralized mortgage obligations, and explain the cash flows and risks for each type.

A **mortgage pass-through security** represents a claim on the cash flows from a pool of mortgages, net of administration fees. Any number of mortgages may be used to form the pool, and any mortgage included in the pool is referred to as a **securitized mortgage**.

The mortgages in the pool typically have different maturities and different mortgage rates. The **weighted average maturity (WAM)** of the pool is equal to the weighted average of the final maturities of all the mortgages in the pool, weighted by each mortgage's outstanding principal balance as a proportion of the total outstanding principal value of all the mortgages in the pool. The **weighted average coupon (WAC)** of the pool is the weighted average of the interest rates of all the mortgages in the pool (weighted in the same way as WAM).

Investors in mortgage pass-through securities receive the monthly cash flows generated by the underlying pool of mortgages, less any servicing and guarantee/insurance fees. The fees account for the fact that **pass-through rates** (i.e., the coupon rate on the MBS, also called its net interest or net coupon) are less than the WAC of the underlying mortgage pool.


<IMG SRC="https://github.com/PachaTech/CFA-Level-1/blob/main/Fixed%20Income/Module%2067/pics/pass.jpeg?raw=true">

<hr>

#### Example: WAM and WAC

*An MBS contains the three mortgages A, B, and C, with details displayed in the following table.*

<table>
<thead>
  <tr>
    <th>Mortgage</th>
    <th>Interest Rate</th>
    <th>Beginning Balance (000 USD)</th>
    <th>Current Balance (000 USD)</th>
    <th>Original Term (Months)</th>
    <th>Months to Maturity</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>A</td>
    <td>2.6%</td>
    <td>100</td>
    <td>90</td>
    <td>240</td>
    <td>210</td>
  </tr>
  <tr>
    <td>B</td>
    <td>1.0%</td>
    <td>200</td>
    <td>72</td>
    <td>300</td>
    <td>100</td>
  </tr>
  <tr>
    <td>C</td>
    <td>5.4%</td>
    <td>300</td>
    <td>247</td>
    <td>360</td>
    <td>280</td>
  </tr>
</tbody>
</table>

<hr>

Calculate the WAM and the WAC for this MBS.

In [15]:
# Calculate from above table the WAM of the MBS
WAM = 210*(90/409) + 100*(72/409) + 280*(247/409)
print("Weighted average maturity WAM in months:",round(WAM,0))

WAC = 0.026*(90/409) + 0.01*(72/409) + 0.054*(247/409)
print("Weighted average coupon WAC in percent:  %",round(WAC,3)*100)

Weighted average maturity WAM in months: 233.0
Weighted average coupon WAC in percent:  % 4.0


#### Answer:

Both the WAM and the WAC are calculated using the current balance of the mortgages as weights. Given that the sum of current balances is `90 + 72 + 247 = 409`, 
\
\
the WAM is calculated as follows:
\
$$
\text{WAM} = 210\Large(\small\frac{90}{409}\Large) \small+ 100 \Large(\small\frac{72}{409}\Large) \small+ 280 \Large(\small\frac{247}{409}\Large) \small = 233 \text{ months}
$$
\
\
The WAC is calculated as follows:
\
$$\text{WAC} = 2.6\% \Large(\small\frac{90}{409}\Large) \small+ 1.0\%  \Large(\small\frac{72}{409}\Large) \small+ 5.4\%  \Large(\small\frac{247}{409}\Large) \small = 4.0\% 
$$

#### Collateralized Mortgage Obligations

**Collateralized mortgage obligations (CMOs)** are securities that are collateralized by pass-through MBSs and pools of mortgages. Each CMO has multiple bond classes (CMO tranches) that have different exposures to prepayment risk. The total prepayment risk of the underlying RMBS is not changed, but is reapportioned among the various CMO tranches.

Institutional investors have different tolerances for prepayment risk. Some are primarily concerned with extension risk, while others may want to minimize exposure to contraction risk. By partitioning and distributing the cash flows generated by RMBSs into different risk packages to better match investor preferences, CMOs increase the potential market for securitized mortgages, and perhaps reduce funding costs as a result.

Common CMO structures include sequential pay tranches, planned amortization class tranches, support tranches, and floating-rate tranches.

In a **sequential pay CMO**, principal payments for the collateral flow to tranches in a prespecified order. Once the first tranche has been fully paid, principal from the underlying collateral will flow to the next tranche in the structure.

As an example, consider a simple CMO with two tranches. Both tranches receive interest payments at a specified coupon rate, but principal payments (both scheduled payments and prepayments) are paid to Tranche 1 (the "short" tranche) until its principal is paid off. Principal payments then flow to Tranche 2 until its principal is paid off.

Contraction and extension risk still exist with this structure, but they have been redistributed to some extent between the two tranches. The short tranche, which matures first, offers investors relatively more protection against extension risk. The other tranche provides relatively more protection against contraction risk.

Investors with preferences for shorter-term securities will prefer tranches paid back sooner because they will have a lower expected average life than other tranches that ranked lower in the order for receiving principal payments.

Other CMO structures include the following:

* **Z-tranches**. A Z-tranche (also called an accrual or accretion bond) is a CMO tranche that receives no interest payments during a specified accrual period. Instead, during the accrual period, the tranche accrues interest as extra principal. For example, if a Z-tranche with a coupon rate of 5% has £100 of principal outstanding at the start of the accrual period, then at the end of the first period, the principal amount of the Z-tranche would be increased to £105 (rather than Z-tranche holders receiving a £5 cash interest payment). Once the accrual payment has ended, interest and principal payments on the Z-tranche begin. A Z-tranche is typically the lowest-ranking tranche in a CMO structure.

* **Principal-only (PO)** securities. These securities pay only principal from the collateral pool; hence, they are effectively zero-coupon securities. Decreasing interest rates and increasing prepayment speeds benefit PO tranche holders because their return comes purely from the difference between the price paid and the repayment of principal. The sooner the principal is repaid, the higher the annualized return to PO security investors.

* **Interest-only (IO) securities**. These securities pay only interest from the collateral pool. Decreasing interest rates and increasing prepayment speeds harm IO tranche holders because their return comes purely from coupon payments on outstanding principal. The sooner the principal is repaid, the fewer coupon payments IO security investors receive.

* **Floating-rate tranches**. These tranches pay a coupon that is linked to a variable market reference rate, often subject to a cap and a floor. Tranches can also be structured as inverse floaters, where the coupon paid rises (falls) as interest rates fall (rise).

* **Residual tranches**. Residual tranches of CMOs rank junior to all other tranches, similar to the equity tranche of an ABS.

* **Planned amortization class (PAC) tranches and support tranches**. A PAC tranche is structured to make predictable payments to investors as long as prepayment speeds remain within a certain range. If prepayment speeds increase, the support tranche receives the principal repayments in excess of those specifically allocated to the PAC tranches. Conversely, if prepayment speeds decrease, principal repayments to the support tranche are curtailed so the scheduled PAC payments can be made. In this way, both contraction and extension risk are reduced for PAC tranche investors. However, this protection is limited by the range of prepayment speeds under which the support tranche can operate. Should speeds move outside this range, the PAC tranche will not receive payments in line with its planned amortization schedule.


### LOS 67.d: Describe characteristics and risks of commercial mortgage-backed securities.

**Commercial mortgage-backed securities (CMBSs)** are backed by pools of commercial mortgages on income-producing real estate, typically in the form of apartments (multifamily), industrial property (e.g., warehouses), shopping centers, office buildings, health care facilities (e.g., senior housing), and hotels.

CMBSs typically have fewer mortgages in the collateral pool than RMBSs—in some cases, a CMBS can even be backed by a single mortgage on a high-value, well-known property in a major city. Hence, there is less diversification against default risk. CMBSs can be backed by mortgages from an individual lender, or multiple commercial mortgages by one borrower. CMBSs are common in the United States, where CMBS coupons are fixed. CMBS volumes are increasing in Europe, where they typically have floating coupons.

An important difference between residential and commercial MBSs is the obligations of the borrowers of the underlying loans. Residential MBS loans are repaid by the owners or occupiers of the property; commercial MBS loans are repaid by real estate investors who, in turn, rely on tenants (usually businesses) of the property and their customers to provide the cash flow to repay the mortgage loan. The regular income from the collateral of a CMBS is referred to as the weighted average proceeds from the mortgages (WAMP), which plays the same role as WAC for an RMBS.

For these reasons, the analysis of CMBSs focuses on the credit risk of the property and not the credit risk of the borrower. Analysts focus on two key ratios to assess credit risk.

1. The **debt service coverage ratio** **(DSCR**, or **DSC ratio)** is a basic cash flow coverage ratio of the amount of cash flow from a commercial property available to make debt service payments, compared to the required debt service cost:

$$
\text{debt service coverage ratio} = \frac{\text{net operating income}}{\text{debt service}}
$$

Net operating income is calculated as rental income less cash operating expenses and a noncash replacement reserve that represents the depreciation of the property over time. This ratio indicates greater protection to the lender when it is higher.

2. The **loan-to-value ratio (LTV)** compares the loan amount on the property to its current fair market or appraisal value:

$$
\text{LTV} = \frac{\text{current mortgage amount}}{\text{current appraised value}}
$$
\
This is the same ratio we discussed in the context of residential mortgages. As was the case there, a higher LTV indicates higher default risk.

Two differences between CMBS and the RMBS we discussed earlier are call protection and balloon maturity provisions.

**Call protection** is equivalent to prepayment protection (i.e., restrictions on the early return of principal through prepayments). CMBSs provide call protection in two ways: loan-level call protection provided by the terms of the individual mortgages, and call protection provided by the CMBS structure.

There are several means of creating **loan-level call protection**:

* *Prepayment lockout*. For a specific time period (typically, two to five years), the borrower is prohibited from prepaying the mortgage loan.

* *Prepayment penalty points*. A penalty fee expressed in points may be charged to borrowers who prepay mortgage principal. Each point is 1% of the principal amount prepaid.

* *Defeasance*. The borrower uses payments in excess of scheduled loan payments to purchase a portfolio of government securities that is sufficient to make the remaining scheduled principal and interest payments on the loan. This protects the lender from prepayments (because they continue to expect to receive the same payments as under the original loan schedule), and it allows the borrower to remove the lender's lien on the property, should they wish to sell the property before the end of the mortgage term.

To create **CMBS-level call protection**, loan pools can be subject to sequential pay tranching in a similar fashion to the process used for CMOs.

Commercial mortgages are typically not fully amortized, meaning that at the end of the loan term, some principal remains outstanding that needs to be paid. This is referred to as a **balloon payment**. If the borrower is unable to arrange refinancing to make this payment, the borrower is in default. This possibility is called **balloon risk**. In this case, the lender might extend the term of the loan during a "workout period" under modified terms. Because balloon risk entails extending the term of the loan, it is a form of extension risk for CMBS investors.

<hr>

## Reading 67: Key Concepts

<hr>

### LOS 67.a

Prepayment risk refers to uncertainty about the timing of the principal cash flows from an MBS. Contraction risk is the risk that loan principal will be repaid more rapidly than expected, typically when interest rates have decreased. Extension risk is the risk that loan principal will be repaid more slowly than expected, typically when interest rates have increased.

Time tranching can be used to distribute prepayment risk across the tranches of an MBS.

### LOS 67.b

The loan-to-value ratio (LTV) indicates the percentage of the value of the real estate collateral that is loaned. Lower LTVs indicate less credit risk.

The debt-to-income ratio (DTI) indicates the size of the monthly debt payments of the borrower relative to their monthly pretax gross income. Lower DTIs indicate lower credit risk.

Agency residential mortgage-backed securities (RMBSs) are guaranteed and issued by the federal government or a government-sponsored enterprise. Mortgages that back agency RMBS must be conforming loans that meet certain minimum credit quality standards. Non-agency RMBSs are issued by private companies and may be backed by riskier nonconforming subprime loans.

### LOS 67.c

Key characteristics of RMBS pass-through securities include the pass-through rate (the coupon rate on the RMBS) and the weighted average maturity (WAM) and weighted average coupon (WAC) of the underlying pool of mortgages.

Collateralized mortgage obligations (CMOs) are collateralized by RMBSs or pools of mortgages. CMOs are structured with tranches that have different exposures to prepayment risks.

In a sequential pay CMO, all scheduled principal payments and prepayments are paid to each tranche in sequence until that tranche is paid off. The first tranche to be paid principal has the most contraction risk, and the last tranche to be paid principal has the most extension risk.

A planned amortization class CMO has PAC tranches that receive predictable cash flows as long as the prepayment rate remains within a predetermined range, and support tranches that have more contraction risk and more extension risk than the PAC tranches.

Other types of CMO tranches include Z-tranches, principal-only tranches, interest-only tranches, floating-rate tranches, and residual tranches.

### LOS 67.d

Commercial mortgage-backed securities (CMBSs) are backed by mortgages on income-producing real estate properties. Because commercial mortgages are nonrecourse loans, analysis of CMBSs focuses on credit risk of the properties. CMBSs are structured in tranches with credit losses absorbed by the lowest-priority tranches in sequence.

Call (prepayment) protection in CMBSs includes loan-level call protection such as prepayment lockout periods, defeasance, prepayment penalty points, and CMBS-level call protection provided by lower-priority tranches.

CMBS loans are more likely to be partially amortizing than residential loans, leading to balloon risk.

Ratios used to analyze the credit risk of a commercial mortgage include the debt service coverage ratio and the loan-to-value ratio.